crates/rslint_lexer/src/lib.rs

```diff
@@ -757,6 +757,32 @@ impl<'src> Lexer<'src> {
         }
     }
 
+    #[inline]
+    fn read_shebang(&mut self) -> LexerReturn {
+        assert_eq!(self.cur, 0);
+        self.next();
+        if let Some(b"#!") = self.bytes.get(0..2) {
+            while self.next().is_some() {
+                let chr = self.get_unicode_char();
+                self.cur += chr.len_utf8() - 1;
+
+                if is_linebreak(chr) {
+                    break;
+                }
+            }
+            tok!(SHEBANG, self.cur)
+        } else {
+            let err = Diagnostic::error()
+                .with_message("Incomplete shebang")
+                .with_labels(vec![
+                    Label::primary(self.file_id, 0..1)
+                        .with_message("Hash is located at the beginning of the file, which is supposed to be interpreted as shebang, but following `!` is not found")
+                ]);
```

Comment:
```suggestion
                .with_message("expected `!` following a `#`, but found none")
                .with_labels(vec![
                    Label::primary(self.file_id, 0..1)
                        .with_message("")
                ]);
```
This would be a bit more consistent with other errors